internal/util/util.go

```diff
@@ -59,6 +59,11 @@ func TimeStampStringPtr() *string {
 	return StrToPtr(time.Now().Format(time.RFC3339))
 }
 
+func ParseTimeStamp(t string) time.Time {
+	parsed, _ := time.Parse(time.RFC3339, t)
```

Comment:
string to time can be fairly error-prone should we return errors so they can be logged?

Reply:
I'm using it only for internally generated strings, so there shouldn't be any errors.  On the other hand, others might use this method for user inputs in the future.  Maybe I should rename it to something like ParseTimeStampIgnoreErrors?

Reply:
how about  MustParseTime and panic on error?

Reply:
Panic for the whole service sounds scary.  I'll just return the error :)